tests/drivers/counter/counter_basic_api/src/test_counter.c

```diff
@@ -797,7 +799,8 @@ static void test_cancelled_alarm_does_not_expire_instance(const char *dev_name)
 
 		k_busy_wait(us/2 + i);
 
-		alarm_cfg.ticks = alarm_cfg.ticks + 2*alarm_cfg.ticks;
```

Comment:
test was suppose to set some alarm in the future (it is cancelled anyway) and for non wrapping (within test) counters this formula was ok but it was actually wrong and initial intention was that it is `2*ticks` (2 ms from now).

Reply:
> and initial intention was that it is `2*ticks` (2 ms from now).

The commit message would be a better place for such additional explanation.